requirements.txt

```diff
@@ -12,7 +12,7 @@ python-novaclient<7.0.0
 python-heatclient
 pywinrm
 requests
-suds
+suds-jurko
```

Comment:
this should either version gated or be matched in the integration tests repo

Reply:
as extra note - in the integration tests repo we need to drop the pshpere dep to facilitate this sanely